lib/bolt/project.rb

```diff
@@ -120,9 +120,16 @@ def validate
 
       %w[tasks plans].each do |conf|
         unless @data.fetch(conf, []).is_a?(Array)
-          raise Bolt::ValidationError, "'#{conf}' in project.yaml must be an array"
+          raise Bolt::ValidationError, "'#{conf}' in bolt-project.yaml must be an array"
         end
       end
     end
+
+    def check_deprecated_file
+      if (@path + 'project.yaml').file? && !load_as_module?
```

Comment:
I think this should warn even if `bolt-project.yaml` is present.

Reply:
My only concern with that is it would warn if a user had a `project.yaml` file in their directory that was being used for something else. Super narrow use case, but I figured if they have a `bolt-project.yaml` then they already know `project.yaml` isn't the right file?

Reply:
Hm, I can see that, but I can also see a repo ending up with both and confusing new users about which is 'right'. If we errored I'd say we shouldn't, but since it's just a warning it's safer to raise it and let the user decide what to do.

Reply:
Fair enough. Updated. :)